.changeset/five-pots-clean.md

```diff
@@ -0,0 +1,6 @@
+---
+'@lion/input-tel': patch
+'@lion/validate-messages': patch
+---
+
+Add replace Min/MaxLength validator messages with Pattern validator message in PhoneNumber validator, since wrong info (param) was shown.
```

Comment:
This should be updated with smth like "removed default Validator message" then (if we remove it)